vendor/github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2017-12-01/compute/availabilitysets.go

```diff
@@ -73,7 +73,7 @@ func (client AvailabilitySetsClient) CreateOrUpdatePreparer(ctx context.Context,
 		"subscriptionId":      autorest.Encode("path", client.SubscriptionID),
 	}
 
-	const APIVersion = "2017-03-30"
+	const APIVersion = "2017-12-01"
```

Comment:
Please deploy the k8s cluster with availability set and make sure word press could be deployed with managed disk.

Reply:
@rjaini

Reply:
Sure. I was doing that validation itself.

Reply:
Scenario is working for latest code with availability set and word press can be deployed with managed disk.